pandas/core/strings.py

```diff
@@ -45,6 +45,15 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
 
     Examples
     --------
+    When ``na_rep`` is `None` (default behavior), NaN value(s)
+    in the Series propagate and return value will be NaN.
+
+    >>> Series(['a','b',np.nan,'c']).str.cat(sep=' ')
```

Comment:
Or as if they weren't present in the original array?

i.e. do a fillna then cat, or dropna then cat?

@jreback